piet-test/src/picture_5.rs

```diff
@@ -21,5 +21,19 @@ pub fn draw(rc: &mut impl RenderContext) -> Result<(), Error> {
     // underline text
     rc.stroke(Line::new((100.0, 52.0), (100.0 + width, 52.0)), &brush, 1.0);
 
+    // get hit test text position and draw a curos
+    // this is for position three, not trailing
+    let hit_test_text_position = layout.hit_test_text_position(3, true);
+
+    if let Some(http) = hit_test_text_position {
+        // TODO should the points be f64? directwrite only returns f32
+        let cursor_x = http.point.x as f64;
+        println!("cursor_x: {}", cursor_x);
+
+        let cursor_brush = rc.solid_brush(Color::rgba8(0x80, 0x80, 0x80, 0xF0));
```

Comment:
The drawing operations can now take `&Color` directly, you don't need to call the solid brush constructor. Probably we didn't update the example code you adapted this from, but for new code it should use the cleaner form.